website/src/wasm-demo/www/index.js

```diff
@@ -0,0 +1,202 @@
+// @ts-check
```

Comment:
since we're running this through webpack anyway, why not use TS? (that shouldn't block this PR though :slightly_smiling_face: )

Reply:
Yeah, that’s actually a really great idea! I’d love to get rid of webpack if possible though, but I guess Monaco needs it?

Reply:
Yeah, TypeScript makes more sense indeed. I just didn't want to add the extra dependencies to ts-loader + extra config required to make TS work for now.
As for webpack, monaco does require webpack-like semantics, such as importing CSS files into JavaScript. This can probably be done with another bundler instead, but probably won't work without any tool (or just tsc).